src/index.js

```diff
@@ -342,7 +342,14 @@ class JsonApi {
   }
 
   modelFor (modelName) {
-    return this.models[modelName]
+    if (modelName) {
+      if (!this.models[modelName]) {
+        throw new Error('API resource definition for model \'' + modelName + '\' not found.')
+      }
+      return this.models[modelName] || {}
+    } else {
+      return {}
+    }
```

Comment:
Two things here:

1. Let's use ES6 [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)
2. Drop the else, as it's not needed

```
modelFor (modelName) {
  if (modelName) {
    if (!this.models[modelName]) {
      throw new Error(`API resource definition for model ${modelName} not found.`)
    }
    return this.models[modelName] || {}
  }
  return {}
}
```

Reply:
Roger that.